modules/mbedtls/Kconfig

```diff
@@ -58,8 +58,6 @@ config MBEDTLS_CFG_FILE
 	  special TLS usage, use available Kconfig options, or select an
 	  alternative config.
 
-rsource "Kconfig.tls-generic"
```

Comment:
There is a dependency to `MBEDTLS_BUILTIN` inside the file, why would it be an issue for nrf_security then? If it is, maybe we need to wrap this line within `if MBEDTLS_BUILTIN`.

Reply:
Comment is outdated, We still require more things inside, so it is still used